app/views/email/PasscodeErrorView.scala.html

```diff
@@ -0,0 +1,36 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+@import views.html.templates.inputs.Text
+
+@this(mainTemplate: MainTemplate, text: Text, form: FormWithCSRF)
```

Comment:
You don't need the `Text` and `FormWithCSRF` injects/imports as those components are not used in this view.